src/layouts/default.hbs

```diff
@@ -0,0 +1,143 @@
+<!DOCTYPE html>
+<html lang="en" class="no-js">
+
+<head>
+	<meta charset="utf-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<!-- Site icons -->
+	<meta name="theme-color" content="#5E3B4D" />
+	<link rel="shortcut icon" href="/static/assets/favicon.svg" type="image/svg+xml">
+	<link rel="icon" sizes="16x16 32x32 64x64" href="/static/assets/favicon.svg">
+	<link rel="manifest" href="/static/assets/manifest.webmanifest">
+	{{{meta}}}
+	<link rel="apple-touch-icon" sizes="180x180" href="/static/assets/apple-touch-icon-180.png" />
+	<link rel="mask-icon" href="/static/assets/safari-pinned-tab.svg" color="#5e3b4d" />
+	<meta name="msapplication-TileColor" content="#5E3B4D" />
+	<meta name="msapplication-config" content="/static/assets/browserconfig.xml" />
+
+	<link href="https://fonts.googleapis.com/css?family=Heebo:400,700|Playfair+Display:700" rel="stylesheet">
+	<link rel="stylesheet" href="{{asset "/static/style.scss"}}">
+	{{{blockContent "head"}}}
+	<script src="https://unpkg.com/scrollreveal@4.0.5/dist/scrollreveal.min.js"></script>
+</head>
+
+<body class="is-boxed has-animations">
+	<div class="body-wrap boxed-container">
+		<header class="site-header text-light">
+			<div class="container">
+				<div class="site-header-inner">
+					<div class="brand header-brand">
+						<h1 class="m-0">
+							<a href="/" class="header-link">
+								<img src="/static/images/logo.svg" alt="Logo" />
+								<img src="/static/images/logo-text.png" alt="Gradebook" />
```

Comment:
We need to exclude this line on the homepage, when there is a hero header.

Reply:
Josh mentioned that and said he liked the double logo. What are your thoughts?

Reply:
Hmm. I think it's too busy. If you look at it on mobile it looks really dumb, because there are two logos of the same size.